ethers-solc/src/project_util/mod.rs

```diff
@@ -34,8 +34,10 @@ pub struct TempProject<T: ArtifactOutput = ConfigurableArtifacts> {
 impl<T: ArtifactOutput> TempProject<T> {
     /// Makes sure all resources are created
     pub fn create_new(root: TempDir, inner: Project<T>) -> std::result::Result<Self, SolcIoError> {
-        let project = Self { _root: root, inner };
+        let mut project = Self { _root: root, inner };
         project.paths().create_all()?;
+        // ignore license warnings
+        project.inner.ignored_error_codes.push(1878);
```

Comment:
Should we have an `enum` for error codes and then push the appropriate variant here?

Reply:
we have this in foundry config, since this here is only used for testing, think it's fine.
but should think about moving the error code enum to ethers-solc instead